src/SpriteAnimation/SpriteAnimation.js

```diff
@@ -493,8 +502,14 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	_generateKeyframe: function (percent, x, y) {
-		return (Math.ceil(percent*1000000) / 10000) +'%	{ -webkit-transform: translate3d('+ x +'px, '+ y +'px, 0);	transform: translate3d('+ x +'px, '+ y +'px, 0); }\n';
+	_generateKeyframe: function (percent, x, y, prefix) {
```

Comment:
Why not just change this method to generate all the prefixed methods at once and only call it once?  Or would that cause too big a change?